src/Serialization/Fields.jl

```diff
@@ -285,7 +285,7 @@ function load_internal(s::DeserializerState,
                        ::Type{<: AbstractAlgebra.Generic.RationalFunctionField},
                        dict::Dict)
     R = load_unknown_type(s, dict[:base_ring])
-    symbols = load_unknown_type(s, dict[:symbols])
+    symbols = load_type_dispatch(s, Symbol, dict[:symbols])
```

Comment:
Hmm, `symbols(R)` for a `RationalFunctionField`, and thus presumably also  `dict[:symbols]` (?) can be either a single `Symbol`, or a `Vector{Symbol}`. This now seems to only support a single `Symbol`? Or am I misunderstanding?

Reply:
sorry, this was my mistake